hi.html

```diff
@@ -1,9 +1,8 @@
 <script>
 	var correctAnswer = Math.floor(Math.random() * 100);
-	var numGuessesTaken = 0;
+	var numGuessesTaken = 1;
 	var gameCompleted = false;
-	alert("Using the clues below, guess the correct number, which is between 0 and 100")
-
+	
 	function getFactors(number) {		// returns array of all factors of a number.  empty array denotes prime number
 		var factors = [];
```

Comment:
overall, the formatting is good, just make sure to use two-spaced, soft tabs. (to make it easier to read on github)